{
    "checksum": "1ff8133b16a26f904bd747abb12ce0a0a3807d497a95e8f7bd5a1084c07ff0b4",
    "file": "pybde_error.h",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-c"
    },
    "number_of_duplicates": 2,
    "package": "libbde",
    "path": "libbde/20190102-2/pybde/pybde_error.h",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/libbde/20190102-2/",
        "metric": {
            "size": 14016
        },
        "pts_link": "https://tracker.debian.org/pkg/libbde",
        "sloc": [
            [
                "ansic",
                214672
            ],
            [
                "sh",
                5968
            ],
            [
                "makefile",
                1412
            ],
            [
                "python",
                434
            ],
            [
                "sed",
                16
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/pkg-security-team/libbde",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/libb/libbde/20190102-2/pybde/pybde_error.h",
    "stat": {
        "perms": "rw-r--r--",
        "size": 1524,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "20190102-2"
}